modules/porous_flow/paper.md

```diff
@@ -23,28 +23,72 @@ date: 13 February 2020
+\end{equation}
+Here $i$ and $j$ are spatial indices; $v_{s}^{j}$ is the $j^{\mathrm{th}}$ component of the velocity $\mathbf{v}_{s}$; $\rho_{\mathrm{mat}}$ is the
+mass-density of the material (including solid and fluid contributions); and $b_{j}$ are the
+components of the external force density (for example, the
+gravitational acceleration).  The effective stress $\sigma^{\mathrm{eff}}$ obeys a constitutive equation, $\alpha_{B}$ is the Biot coefficient, and $P_{f}$ an effective fluid porepressure.
+
+A comprehensive listing of equations, including fluid equations of state, the evolution of porosity and permeability, relative-permeability functions, capillary relationships, chemical reactions, etc, may be found at https://mooseframework.org/modules/porous_flow/index.html.
+
+# Software details and comparisons
+
+``PorousFlow`` is built upon the open-source, massively parallel, fully implicit multiphysics simulation framework MOOSE (Multiphysics Object-Oriented Simulation Environment) [@permann2020moose].  MOOSE is an open-source library from Idaho National Laboratory that provides a high-level interface to the libMesh finite element library [@libmesh] and PETSc nonlinear solvers [@petsc].  MOOSE and ``PorousFlow`` follow [strict quality controls](https://mooseframework.org/sqa/index.html).
+
 Other open-source codes that can also perform tightly-coupled THMC simulations to some degree include OpenGeoSys [@opengeosys], PFLOTRAN [@pflotran] and work under the Open Porous Media initiative [@opm].
 
-``PorousFlow`` is built upon the open-source, massively parallel, fully implicit multiphysics simulation framework MOOSE (Multiphysics Object-Oriented Simulation Environment) [@permann2019moose].  MOOSE is an open-source library from Idaho National Laboratory that provides a high-level interface to the libMesh finite element library [@libmesh] and PETSc nonlinear solvers [@petsc].  As such, it runs well on the smallest notebook computer through to the world's largest supercomputers.  MOOSE and ``PorousFlow`` follow [strict quality controls](https://mooseframework.org/sqa/index.html).
+Harnessing the power of MOOSE means that ``PorousFlow`` simulations efficiently use multiprocessors and automatically utilize threading/OpenMP, without developers or users having to worry about the software technicalities.  Mesh adaptivity with a variety of integration schemes and unstructured meshes may be used.  Various time-stepping schemes may be employed.  A variety of I/O formats are available, such as Exodus and VTK files as well as CSV plaintext.  The flexible plug-in architecture makes it easy for novice developers to solve new physics equations or couple together existing modules to solve complex problems.  Users may even utilising the "MultiApp" system that allows simulations to be embedded inside other simulations.  A GUI helps users create, run and analyse models.  An example ``PorousFlow`` result is shown in \autoref{fig:example}.
```

Comment:
`Users may even utilising the` -> `Users may utilise the` perhaps?

I guess no need to expand on I/O or gui as the readership is targeted towards software users.

Reply:
Thanks, yep, i agree